test/extensions/filters/network/common/redis/codec_impl_test.cc

```diff
@@ -65,6 +66,18 @@ TEST_F(RedisEncoderDecoderImplTest, SimpleString) {
   EXPECT_EQ(0UL, buffer_.length());
 }
 
+TEST_F(RedisEncoderDecoderImplTest, BulkString) {
```

Comment:
Was this test missing? Why was this added?

Reply:
This test is new- since a bunch of the other types are there, I figured I'd add it. In theory the array tests capture it; I think one could probably remove some of the other types and do them in the array tests too, but it reads nicer. I can remove if you prefer.

Reply:
Nope thanks for adding, just curious.